dtfs-central-api/src/services/state-machines/fee-record/event-handlers/generate-keying-data/generate-keying-data.event-handler.test.ts

```diff
@@ -311,7 +312,16 @@ describe('handleFeeRecordGenerateKeyingDataEvent', () => {
         end: { month: 5, year: 2025 },
       };
 
+      const ukefShareOfUtilisation = 78787.87;
       jest.mocked(calculateUkefShareOfUtilisation).mockReturnValue(78787.87);
+      const { coverEndDate, interestPercentage, dayCountBasis } = keyingSheetCalculationTfmFacilityValues;
+      const fixedFee = calculateFixedFee({
```

Comment:
Can we add some new lines after the const's etc please

Reply:
done